rmw_connext_cpp/src/functions.cpp

```diff
@@ -1829,6 +2180,7 @@ rmw_service_server_is_available(
     number_of_response_publishers);
 #endif
   if (number_of_response_publishers == 0) {
+    fprintf(stderr, "Number of response publishers is 0\n");
```

Comment:
This gets printed in normal use cases... https://github.com/ros2/rmw_connext/pull/322